html/forms/form-validation/full-example.html

```diff
@@ -36,16 +36,14 @@
 
 <body>
   <form>
-    <p>
-      <fieldset>
-        <legend>Do you have a driver's license?<span aria-label="required">*</span></legend>
-        <!-- While only one radio button in a same-named group can be selected at a time,
-             and therefore only one radio button in a same-named group having the "required"
-             attribute suffices in making a selection a requirement -->
-        <input type="radio" required name="driver" id="r1" value="yes"><label for="r1">Yes</label>
-        <input type="radio" required name="driver" id="r2" value="no"><label for="r2">No</label>
-      </fieldset>
-    </p>
+    <fieldset>
```

Comment:
`<p>` elements can have only phrasing content.
Browsers treat current code as follows:
```html
<p></p>
<fieldset>...</fieldset>
<p></p>
```

Reply:
Is there a change in mdn/content that has to be done?

Reply:
> Is there a change in mdn/content that has to be done?

Yes, https://github.com/mdn/content/pull/30901